### DISCUSSION TITLE
mysql adapter in v0.11.1 looks to be broken

### DISCUSSION THREAD

**Discussion by tuxaanand** [imdrasil/jennifer.cr#393] [General]
Hi All,

I'm trying out jennifer and while executing the sam CLI, I got the below error.

> 
> ➜  kemal101 git:(master) ✗ crystal sam.cr -- help
> Showing last frame. Use --error-trace for full trace.
> 
> In lib/jennifer/src/jennifer/adapter/mysql.cr:10:5
> 
>  10 | class Adapter < Adapter::Base
>       ^
> Error: abstract `def Jennifer::Adapter::Base#insert(obj : Model::Base)` must be implemented by Jennifer::Mysql::Adapter
> 
> 

Looking at the mysql adapter I found that the insert method (https://github.com/imdrasil/jennifer.cr/blob/v0.11.1/src/jennifer/adapter/mysql.cr) is not implemented (the same is available in the master via `include ::Jennifer::Adapter::RequestMethods`)

This may be a bug in `0.11.1` I believe. Please correct me if I'm wrong.

Thanks,
Aanand

**Comment by imdrasil** [comment]
Yes, this issue is caused by changes from the latest crystal version. Right now `master` branch has a fix for it. When I complete last PR and merge it in I'll make a new release

**Comment by imdrasil** [comment]
it should be fixed
